governance/tech-lead-proposal-process.md

```diff
@@ -0,0 +1,107 @@
+<!-- cSpell:ignore chairpersonship markdownlint -->
+# Security TAG Technical Lead Proposal Process
+
+Security Technical Leads operate as a team alongside the Co-Chairs. The Co-Chair
+team seeks enhance the leadership capabilities of the existing co-chairs across
+the key qualities of industry experience, hands-on cloud-native and security
+experience, as well as administrative experience needed to assist in the
+execution of the various projects and efforts that make the Security TAG so
+successful.
+
+## Caveats of Technical Leads
+
+* The position of a technical lead does not guarantee future co-chairpersonship
+* The position of a technical lead is not a paid position (co-chairs are also
+  not paid positions)
+* Technical leads [have responsibilities](roles.md) to the Security TAG, the
+  leadership team, and the TOC
+* Nominations by the community (with co-chair endorsement) or co-chairs still
+  [requires TOC
+  approval](https://github.com/cncf/toc/blob/main/tags/cncf-tags.md#elections)
+
+## Process of nominations
+
+Depending on the needs of the community, as determined by the Security TAG
+leadership team, technical lead nominations may come in the form of "Nominees by
+community", "Nominees by co-chairs", or both.
+
+The Security TAG does not have a current limit on the amount of technical leads.
+
+Final technical lead nominations will adhere to the [CNCF's TOC described
+process](https://github.com/cncf/toc/blob/main/tags/cncf-tags.md#elections).
+
+### Nominees by community
```

Comment:
probably will need a mention on who can nominate, can be a fairly low bar, but probably good to have some baseline so we know we are getting valid and non-repeat entries, e.g. part of CNCF slack TAG-Security channel

Reply:
Another question: will nominations be public or private? I think private would be better for nominations (so folks are more comfortable nominating), but we would release aggregate data about the nominees at the end (without who nominated them, just numbers).